doc/md/stablememory.md

```diff
@@ -1,61 +1,61 @@
 # Stable Memory
```

Comment:
It looks like you've edited the original doc for experimental stable memory. I guess we'll need to keep both, so will need to restore the original and add new one for Regions.md

Reply:
Ditto for the example it references.

Reply:
👀